### DISCUSSION TITLE
ReferenceError: Response is not defined

### DISCUSSION THREAD

**Discussion by jvskriubakken** [nrwl/nx#7580] [Show and Tell]
Hi,

I am having some tests that mocks the fetch api and returns a response object. 

However these tests fails with:
```
 ReferenceError: Response is not defined

      711 |         .mockImplementationOnce(() => {
      712 |           return Promise.resolve(
    > 713 |             new Response(JSON.stringify({ someProp: "x" }), { status: 200 })
          |                 ^
      714 |           );
      715 |         })
      716 |         .mockImplementationOnce(() => {


```

Googling this, brings me to the following jest issue:
https://github.com/facebook/jest/issues/7007

Where Jest says: "We do not polyfill your environment for you"

So my question is... does Nx have a better, solution than having to `require("isomorphic-fetch")` in every test file?

**Comment by avchugaev** [comment]
@jvskriubakken this does not seem related to NX. You can as you question on StackOverflow.
